src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -363,8 +371,8 @@ func NewBusyboxPodWithName(namespace, podName string) PodOrError {
 	return fromPod(NewPod(namespace, podName, "busybox", busyboxImage, WithCommand([]string{"/bin/sh", "-c", "sleep 3600"})))
 }
 
-func NewBusyboxPodWithNameWithInitdata(namespace, podName string, kbsEndpoint string) PodOrError {
-	initdata := fmt.Sprintf(testInitdata, kbsEndpoint, kbsEndpoint, kbsEndpoint)
+func NewBusyboxPodWithNameWithInitdatahttps(namespace, podName string, kbsEndpoint string) PodOrError {
```

Comment:
nit: I think we decided that only testing HTTPS was the way forward, so I don't think we need to rename this function if we don't have an http version?